components/Layout.tsx

```diff
@@ -1,4 +1,4 @@
-import Meta from '@components/page/Meta';
+import Meta from '@components/Meta';
```

Comment:
i changed it because i don't think it makes sense to put it in the folder for the devfile page when it also applies to the home-page 
i can also create a different folder for it but personally i think it not being in a subdirectory of components makes sense